### PR TITLE
add instructions to set max per-locale CPU cores and memory

### LINKED ISSUE #2418: better document options for limiting per-locale memory
**Feature Description**
There are two means of limiting per-locale memory, one by pct of total physical memory on host machine, one via the Chapel memMax command-line argument. The Arkouda docs should discuss both to ensure users are aware of both approaches.

**Additional context**
The Arkouda docs will be updated to fully document both approaches to limiting per-locale memory

### PR DESCRIPTION
**Pull request by hokiegeek2**
Closes #2418 

This PR adds content to the README file that details how to set per-locale limits for memory and CPU cores utilized.

@21771 this is per our discussion last week

### CODE REVIEW

**Review by Ethan-DeBandi99 — APPROVED** [review]
Looks good to me!

**Review by pierce314159 — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -251,7 +250,30 @@ tokens.txt file.
...
+
+````
+./arkouda_server --perLocaleMemLimit=70
+```
+
+In addition, the max per-locale memory can instead be set to an explicit number of bytes via the --memMax startup parameter. For example, to set the max memory utilized by each locale to 100 GB, the Arkouda startup command would be as follows:
+
+````
+./arkouda_server --memMax=100000000000
+````
```
[the remark this answers is not in the document]
Comment (hokiegeek2) [reply README.md:274]: @pierce314159 argh, okay, fixing now

[the remark this answers is not in the document]
Comment (hokiegeek2) [reply README.md]: @pierce314159 fixed! thanks for finding this error

**Review by pierce314159 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -251,7 +250,30 @@ tokens.txt file.
...
+The max memory utilized by each locale can be set in one of two ways: percentage of physical memory or a limit set in bytes. By default, the max per-locale memory is set to ninety (90) percent of the physical memory on each Arkouda locale host. If another percentage is desired, this is set via the --perLocaleMemLimit startup parameter. For example, to set max memory utilized by each locale to seventy (70) percent of physical memory, the Arkouda startup command would be as follows:
+
+```
+./arkouda_server --perLocaleMemLimit=70
+```
+
+In addition, the max per-locale memory can instead be set to an explicit number of bytes via the --memMax startup parameter. For example, to set the max memory utilized by each locale to 100 GB, the Arkouda startup command would be as follows:
+
+```
+./akouda_server --memMax=100000000000
```
Comment (pierce314159) [inline README.md:273]:
```suggestion
./arkouda_server --memMax=100000000000
```

Comment (pierce314159) [reply README.md:273]: Sorry i didn't see this until after i reviewed but we dropped an `r` in `arkouda` somehow

**Review by pierce314159 — APPROVED** [review]

**Pull request merged**